container/common/helpers.go

```diff
@@ -258,6 +258,13 @@ func assignDeviceNamesToPerDiskStats(namer DeviceNamer, diskStats ...[]info.PerD
 	}
 }
 
+// FilterCgroupPaths delete cgroup paths which we do't need base on includeMetrics
+func FilterCgroupPaths(cgroups map[string]string, includedMetrics container.MetricSet) {
```

Comment:
rather than filtering out afterwards, I think it would be cleaner to do this when we first get the list of cgroup subsystems: https://github.com/google/cadvisor/blob/master/container/libcontainer/helpers.go#L37.